apps/charterafrica/src/pages/api/v1/opportunities/consultation/[...service].page.js

```diff
@@ -0,0 +1,49 @@
+const documents = async (req, res) => {
+  const baseURL = "https://dc.sourceafrica.net/api/oembed.json";
+  const { url } = req.query;
+  try {
+    const fullURL = `${baseURL}?url=${url}`;
+    const response = await fetch(fullURL);
+    const data = await response.json();
+    res.status(200).json(data);
+  } catch (error) {
+    res.status(500).json({ error });
+  }
+};
+
+const youtube = async (req, res) => {
+  try {
+    const { service, ...rest } = req.query;
+    const url = service?.slice(1).join("/");
+    const key = process.env.GOOGLE_MAPS_API_KEY;
+    const queryParams = { key, ...rest };
+    const queryString = new URLSearchParams(queryParams).toString();
+    const youtubeBaseUrl = `https://www.googleapis.com/youtube/v3`;
+    const response = await fetch(`${youtubeBaseUrl}/${url}?${queryString}`);
+    const data = await response.json();
+    res.status(response.status).json(data);
+  } catch (error) {
+    res.status(500).json(error);
+  }
+};
+
+const serviceMap = {
+  documents,
+  youtube,
+};
```

Comment:
```suggestion
const mediaMap = {
  documents,
  multimedia,
};
```

or audiovideo/av instead of multimedia.